classes/03class/exercises/c03-docker06/mail2kvanitha/Dockerfile1

```diff
@@ -0,0 +1,3 @@
+FROM alpine:latest
```

Comment:
using latest is not a best practice as a change in the upstream may affect your work. can you use a specific version of alpine?